evaluate/task/transpile.go

```diff
@@ -68,7 +68,7 @@ func (t *Transpile) Run(ctx evaltask.Context) (repositoryAssessment map[string]m
 
 		modelAssessments := metrics.NewAssessments()
 		withSymflowerAssessments := metrics.NewAssessments()
-		maximumReachableFiles := uint64(len(language.Languages) - 1) // Transpile repositories contain sub-tasks to transpile from every other supported language minus the one we are transpiling to.
+		maximumReachableFiles := uint64(len(originFilePathsWithLanguage))
```

Comment:
@zimmski Maybe this is to blame for the wrong maximum?

Reply:
The problem is with write-tests task. So unlikely. but can you check if this change is correct?

Reply:
change is legit